services/iaasalpha/wait/wait.go

```diff
@@ -346,3 +349,53 @@ func DeleteVirtualIPWaitHandler(ctx context.Context, a APIClientInterface, proje
 	handler.SetTimeout(15 * time.Minute)
 	return handler
 }
+
+// ImageUploadWaitHandler will wait for server creation
```

Comment:
We should add more info here on the process and what the function is actually checking: status AVAILABLE that indicates the upload of the image was successful